src/AstroBase.jl

```diff
@@ -115,4 +115,64 @@ function tio_locator(jd1, jd2)
     -47e-6 * t * sec2rad(1)
 end
 
+"""
+    precession_rate_part_of_nutation(jd1, jd2)
+
+Returns precession corrections for a given 2 part Julian date (TT).
+
+# Example
+
+```jldoctest
+
+```
+"""
+function precession_rate_part_of_nutation(jd1, jd2)
+    PRECESSION = -sec2rad(0.29965)
+    OBLIQUITY = -sec2rad(0.02524)
+
+    t = ((jd1 - J2000) + jd2) / DAYS_PER_CENTURY
+    PRECESSION * t, OBLIQUITY * t
+end
+
+function bias_component_00()
```

Comment:
Does this need to be a function at all?

Reply:
No, we can essentially put it in constants and access the variables from there. It's our call, though the function makes it neat.